docker/worker/Dockerfile

```diff
@@ -66,6 +68,12 @@ RUN cp /home/turbinia/password.lst /root/
 
 RUN pip3 install impacket --no-deps
 
+RUN cd /home/turbinia && mkdir Loki && cd Loki \
+    && curl -s https://api.github.com/repos/Neo23x0/Loki/releases/latest | sed -n 's/.*"tarball_url": "\(.*\)",.*/\1/p' | xargs -n1 wget -O - -q | tar -xz --strip-components=1 \
+    && pip3 install -r requirements.txt \
+    && sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
```

Comment:
This retreives the latest release - it changed while I was developing and all my stuff broke so this little snippet always gets the most recent

Reply:
Also yes it hardcodes /usr/bin/python :-/

Reply:
Can all the pip install pacakge stuff be moved above line 50 where we move ./tmp into the build container. If we start installing python packages after that it builds become very slow due to invalidated layers. Eg if you change 1 file in the Turbinia source it should be a simple copy file layer, but in this case it will also re-install the python packages.

Reply:
Great point, done.